geomesa-quickstart-accumulo/src/main/java/com/example/geomesa/accumulo/AccumuloQuickStart.java

```diff
@@ -347,5 +360,23 @@ public static void main(String[] args) throws Exception {
                 "What");
 
         dataStore.dispose();
+
+        if(cmd.hasOption(DELETE_TABLE)) {
```

Comment:
can we cast to AccumuloDataStore and use the `delete` method instead?
https://github.com/locationtech/geomesa/blob/master/geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala#L71
The _queries table will not be deleted, so you would still need to do that manually. You already have access to the `connector` object from the datastore, so you can use the `tableOperations` method on that to get a handle on the table ops instead of re-creating it.
The problem with hard-coding the table names here is that they are likely to change eventually - using the data store method will ensure that if they do, they will still be deleted.